Remotion/ObjectBinding/Core/BindableObject/Properties/DateOnlyProperty.cs

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+
+namespace Remotion.ObjectBinding.BindableObject.Properties
+{
+#if NET6_0_OR_GREATER
+  public class DateOnlyProperty : DateTimePropertyBase
+  {
+    public DateOnlyProperty (Parameters parameters)
+        : base(parameters)
+    {
+    }
+
+    public override object? ConvertFromNativePropertyType (object? nativeValue)
+    {
+      if (nativeValue != null)
+        return ((DateOnly)nativeValue).ToDateTime(TimeOnly.MinValue);
+      return base.ConvertFromNativePropertyType(nativeValue);
```

Comment:
the other proprties use
```suggestion
      var dateOnly = ArgumentUtility.CheckType<DateOnly?>("nativeValue", nativeValue);
      if (dateOnly != null)
        return (dateOnly.ToDateTime(TimeOnly.MinValue);
      return base.ConvertFromNativePropertyType(dateOnly);
```